projects/fsastorefrontlib/src/cms-components/my-account/documents/documents-overview/documents-overview.component.html

```diff
@@ -1,9 +1,18 @@
 <ng-container *ngIf="documentList$ | async as documentList; else loading">
   <!-- ng-container needs check if documents are uploaded by user or they came from external system -->
   <ng-container *ngIf="(documentList | json) != '{}'; else noDocuments">
-    <cx-fs-documents-table
-      [documentSource]="documentList?.documents"
-    ></cx-fs-documents-table>
+    <ng-container *ngIf="checkDocumentsSource(documentList?.documents, false)">
```

Comment:
Could you use the same method getDocumentsBySource here and just check if array of documents is not empty?

Reply:
Yes, it can be used in that purpose. Fixed!